packages/xo-server/src/xo-mixins/jobs/index.mjs

```diff
@@ -173,7 +173,6 @@ export default class Jobs {
       data:
         type === 'backup' || type === 'metadataBackup' || type === 'mirrorBackup'
           ? {
-              ignoreSkippedBackups: job.settings['']?.ignoreSkippedBackups,
```

Comment:
Should we keep supporting `ignoreSkippedBackups` for users who are using it and haven't edited their backup jobs, or you think it's not necessary?

Reply:
I don't think it's necessary, it's been there for only a week and has not been part of a commercial release.
And for the potential users who started to use it, the only consequence is that they may receive again reports for skipped backups before modifying their backup job configuration